TombEngine/Game/Lara/lara_one_gun.cpp

```diff
@@ -556,109 +562,99 @@ void UndrawShotgunMeshes(ItemInfo& laraItem, LaraWeaponType weaponType)
 	}
 }
 
-ItemInfo* FireHarpoon(ItemInfo& laraItem)
+void FireHarpoon(ItemInfo& laraItem, Pose* pos)
```

Comment:
I would recommend using `std::optional<Pose>` here instead of a `Pose*` pointer.